### PR TITLE
Added link to a question

### PR DESCRIPTION
**Pull request by alearanibar**
Added css for anchor link

### CODE REVIEW

**Comment by andrewevans** [comment]
Nest your branch within one of the current groups (feature, fix, infra, design, refactor), followed by a descriptive name of the branch. For example, this one should be something like feature/question-title-html. Also, since this specific link isn't necessarily part of the question being asked, we may want to refactor this to use the "description" property in the future.

File: app/styles/app.less [hunk 1]
```diff
@@ -103,6 +103,10 @@
     padding-left: (32px + 12px);
     font-family: arial;
     font-weight: normal;
+    a {
+       color: #ffffff;
+       text-decoration: underline;
```
Comment (andrewevans) [inline app/styles/app.less]: CS. Remove extra whitespace at EOL

File: app/styles/app.less [hunk 2]
```diff
@@ -103,6 +103,10 @@
     padding-left: (32px + 12px);
     font-family: arial;
     font-weight: normal;
+    a {
```
Comment (andrewevans) [inline app/styles/app.less]: Add newline above definition.

File: mirage/scenarios/default.js [hunk 3]
```diff
@@ -79,7 +79,7 @@ export default function(server) {
 
   // Create a question + its options
   question = chapter.createQuestion({
-    title: "Congratulations! How many weeks pregnant are you?",
+    title: "Congratulations! How many weeks pregnant are you? Not sure? <a target='_blank' href='/pregnancy-due-date-calculator'>Check with our due date calculator.</a>",
```
Comment (andrewevans) [inline mirage/scenarios/default.js]: Do you suggest we make using single quotes within a property's strings part of our coding standards? Do you think it's better than escaping double quotes?

**Comment by andrewevans** [comment]
Try to keep updates to the mock API data in separate commits from changes to the app. It helps act as if the mock API is handled by a separate developer and can change somewhat independently of the app. For example, in this PR, the `default.js` change could go in a separate commit after the app changes, with commit message "Mock data." No need to change it for this PR, though.

**Comment by alearanibar** [comment]
All changes are on the new branch https://github.com/andrewevans/summer/pull/74

**Comment by andrewevans** [comment]
This PR is closed because the changes here are reflected in PR 74 .

**Pull request closed without merging**